parquet/src/main/java/org/apache/iceberg/parquet/ParquetUtil.java

```diff
@@ -222,6 +222,24 @@ private static MessageType getParquetTypeWithIds(
     return ParquetSchemaUtil.addFallbackIds(type);
   }
 
+  /**
+   * Returns a list of offsets in ascending order determined by the starting position of the row
+   * groups.
+   */
+  public static List<Long> getSplitOffsets(InputFile file) {
```

Comment:
What's the use case that benefits from this public method? I think we only get offsets in writers at the moment and we always have `ParquetMetadata` at that point.

Reply:
I'm adding files directly to a table, that isn't using a writer. 
I already have existing Parquet files on S3.

To add those files I'm using `DataFile.builder()` combined with `table.newAppend().appendFile().commit()`.  The problem is by not using a writer there is no way to obtain the split offsets without importing the Hadoop dependency to read the split offsets.  By adding this method I can avoid importing the Hadoop dependency saying about 300mb in a JAR file.